src/contractTest/java/uk/gov/hmcts/reform/orgrolemapping/provider/OrgRoleMappingProviderTestForRefresh.java

```diff
@@ -80,6 +80,7 @@ void pactVerificationTestTemplate(PactVerificationContext context) {
     @BeforeEach
     void beforeCreate(PactVerificationContext context) {
         MockMvcTestTarget testTarget = new MockMvcTestTarget();
+        //System.getProperties().setProperty("pact.verifier.publishResults", "true");
```

Comment:
Can this be removed? If not shall we add a comment so others know why it is commented out?

Reply:
further comment added with reason for inclusion. resolved